test_pgeocode.py

```diff
@@ -26,6 +26,11 @@ def temp_dir():
     shutil.rmtree(path)
 
 
+@pytest.fixture(scope='session')
+def country_gb_full():
+    return pgeocode._Country('gb_full')
```

Comment:
Typically session level fixtures are used for expensive operation (e.g. loading the data, or file access operations) once per session run.

To test `_Country` we should rather initialize it in each test that needs (and not use fixture).